drizzlepac/haputils/analyze.py

```diff
@@ -532,7 +552,10 @@ def analyze_data(input_file_list, log_level=logutil.logging.DEBUG, type=""):
             # processing should be allowed, but there may be some issue with the result (e.g.,
             # GYROS mode so some drift)
             generate_msg(input_file, msg_type, no_proc_key, no_proc_value)
-        
+        elif non_zero_data_in_array==False:
+            do_process=False
```

Comment:
add here:
process_msg="SCI data all zeros"
so the message also is put into the output table.

Reply:
Added.